providers/implementations/ciphers/cipher_aes_ocb.c

```diff
@@ -439,7 +439,8 @@ static const OSSL_PARAM cipher_ocb_known_gettable_ctx_params[] = {
     OSSL_PARAM_octet_string(OSSL_CIPHER_PARAM_AEAD_TAG, NULL, 0),
     OSSL_PARAM_END
 };
-static const OSSL_PARAM *cipher_ocb_gettable_ctx_params(void *provctx)
+static const OSSL_PARAM *cipher_ocb_gettable_ctx_params
+    (ossl_unused void *provctx)
```

Comment:
We're starting to see two styles to break these sorts of lines:

1.  ``` C
    static const OSSL_PARAM *
    cipher_ocb_gettable_ctx_params(ossl_unused void *provctx)
    ```
2.  ``` C
    static const OSSL_PARAM *cipher_ocb_gettable_ctx_params
        (ossl_unused void *provctx)
    ```

We really need to decide...
(the former is easier on style helper software, it seems, and also more recognised from elsewhere, as that's the GNU style)

Reply:
(re style helper software, we just need to look at the coloring github provided :wink:)

Reply:
I quite dislike the former ...

Shortening the `provctx` name helps.

Reply:
I still like this more..
3.
```
static const OSSL_PARAM *cipher_ocb_gettable_ctx_params(
    ossl_unused void *provctx)
```

Reply:
Personal opinion on 3: ewwwww

Reply:
Three options, three opinions, all different 😁

Reply:
I dont have the right emoji to print here lots of times..

Reply:
1. is really common. I've almost never seen 2. and I seriously dislike it. 3. would be OK with me.
Proposal 4 - do not break the line earlier than after the first argument even if it overflows the limit.

Reply:
5. Stop using an archaic line limit. I don't think this improves readability with the amount of line wrapping that occurs.

Reply:
100 chars has been proposed as a line limit.  I'd be fine with that.
(I can even do 132 char, which was the numbers of characters on a VT100 in condensed mode, but I frankly find that excessive)